app/res/layout/scrolling_calendar_widget.xml

```diff
@@ -134,12 +134,10 @@
             android:layout_centerVertical="true"
             android:layout_alignParentStart="true" />
 
-        <Button
+        <com.google.android.material.button.MaterialButton
```

Comment:
we need a screenshot to review. This can be accessed by [configuring a calendar on a date widget](https://dimagi.atlassian.net/wiki/spaces/commcarepublic/pages/2143957035/Date+and+Calendar+Display#Use-of-the-Alternate-Calendar-Systems-in-CommCare-Android).

Reply:
![calender ss](https://github.com/user-attachments/assets/08938797-2fe2-49e6-ac36-46a8667a4acb)

Reply:
The [calendar button](https://github.com/dimagi/commcare-android/blob/master/app/res/drawable/icon_calendar_circle_turquoise.xml) only appears for the Gregorian widget, you will need to set the appearance attribute as `gregorian` on your date widget to be able to see it.